CakePHP/Sniffs/Formatting/BlankLineBeforeReturnSniff.php

```diff
@@ -63,10 +63,15 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
         ) {
             return;
         } elseif (count($prevLineTokens) > 0) {
-            $phpcsFile->addError(
+            $fix = $phpcsFile->addFixableError(
                 'Missing blank line before return statement',
                 $stackPtr
             );
+            if ($fix === true) {
```

Comment:
My bad!  I was on mobile and responded from memory instead of actually re-reading the issue.